components/src/core/components/TableFilter/TableFilter.vue

```diff
@@ -122,13 +122,23 @@ export default defineComponent({
 .oxd-table-filter {
   .oxd-table-filter-header-options {
     & .oxd-icon-button,
-    .oxd-button {
-      margin-left: 5px;
+    .oxd-button,
+    .oxd-autocomplete-search-wrapper {
+      margin-left: 0.5rem;
```

Comment:
shall we move this to a sass variable. i imagine other places also have this button gap